components/docs/Sidebar/Dropdown.jsx

```diff
@@ -36,12 +36,14 @@ export default function Dropdown({
         {routes.routes.map((r, idx) => {
           if (!r.path) {
             return (
-              <Dropdown
-                routes={r}
-                parentOpen={open}
-                key={`${r.title}-${idx}`}
-                setSidebarCollapsed={setSidebarCollapsed}
-              />
+              <li key={`${r.title}-${idx}`}>
+                  <Dropdown
+                    routes={r}
+                    parentOpen={open}
+                    key={`${r.title}-${idx}`}
+                    setSidebarCollapsed={setSidebarCollapsed}
+                  />
+              </li>
```

Comment:
note: html requires that each element in a list be an `<li>`, so a Dropdown isn't valid on its own and needs to be wrapped. This looks and works the same as far as I can tell.